Strategies/ITVision.md

```diff
@@ -0,0 +1,168 @@
+
+**Business Rules are seen as institutional knowledge** and are managed [internally by ESDC's staff](https://18f.gsa.gov/2018/10/09/implementing-rules-without-rules-engines/), outside of proprietary systems to avoid vendor lock-in and assure long term sustainability of Canadians core benefits systems.
+
+## 3. Digital-first and digitally enabled by design
+
+All of [ESDC's services](https://www.canada.ca/en/employment-social-development/corporate/portfolio/service-canada/programs.html) are delivered through technology with a large majority running on the public cloud to seek benefits of [rapid elasticity, broad internet access, and up to date security patching](https://doi.org/10.6028/NIST.SP.800-145). This includes up to Protected B data.
+Platform as a Service (PaaS, such as containers) and Software as a Service (SaaS) delivery cloud models ia preferred, leaving Infrastructure as a Service (IaaS) on exception basis only.
+
+ESDC's IT Solutions can only "talk" to each other via **APIs** allowing channel-agnostic services and communications.
+ESDC's API are all published on the Government of Canada's **API Store**, enabling the private industry and the public to build mobile applications, websites, applications, and voice assistant compatibility devices, to deliver a variety of user experiences to citizens.
+
+ESDC's IT solutions provide the flexibility needed to empower end-users with the choice of sharing personal data, bringing any privacy concerns in the hands of the end-user as opposed to ESDC making decisions for them.
+
+ESDC prefers **open source software** and products that adopt **open standards** over proprietary ones.
+The adoption of Open source software is the main vendor locking mitigation strategy.
+The adoption of open standards is the main interoperability enabler strategy.
+
+Legacy technologies like mainframe COBOL have been replaced by using a strangled approach as opposed to a big bang one.
+The transition of legacy systems into modern ones required large investments on the business and data side, as opposed to IT, in order to accurately depict business rules and processes agnostically of IT systems.
+
+ESDC's legacy core systems are modernized into a set of micro services as opposed to monolithic systems, each working together as an aggregate to bring Pension, Employment Insurance, and Old Age Security delivery to citizens.
+The number of services (or number of systems) is not a concern, the concern is rather the ability to effectively manage technical debt and keep up with demands by making timely, small iterative changes.
+
+ESDC leverages different networks based on its need.
+The Internet for its most common day to day use, reducing bottleneck to its corporate and production network.
+The corporate network for accessing corporate systems (e.g. HR, Finance, and emails), and production network for accessing the systems hosting Canadians' data and proving services to Canadians.
```

Comment:
production network

Wouldn't call it the production network ... 

Corporate network and non-corporate network (as in, send YouTube to the internet the same way you would at home). @e-wu Could confirm the best wording, I would think

Reply:
Not sure a distinction between "corporate" and "production" network is necessary. It's all an internal network at the end. If kept, I would indeed change production to something else as the corporate network is by definition a production one ;)

Reply:
I think the idea was to segregate where you do corporate work and where the production systems runtime actually happens.

Up for discussion with @e-wu I think.